--tests--/types.test.js

```diff
@@ -14,6 +22,58 @@ describe('validator module', () => {
       expect(isNumber(() => {})).toBeFalsy();
       expect(isNumber(true)).toBeFalsy();
     });
+    it('properly tells if a value is a string', () => {
+      expect(isString('a string')).toBeTruthy();
+      expect(isString('')).toBeTruthy();
+      expect(isString('double quote string')).toBeTruthy();
+      expect(isString(5)).toBeFalsy();
+      expect(isString(false)).toBeFalsy();
+      expect(isString({})).toBeFalsy();
+      expect(isString([])).toBeFalsy();
+      expect(isString(()=>{})).toBeFalsy();
+    });
+    it('properly tells if a value is a boolean', () => {
+      expect(isBoolean(true)).toBeTruthy();
+      expect(isBoolean('hi')).toBeFalsy();
+      expect(isBoolean('true')).toBeFalsy();
+      //expect(isBoolean("")).toBeFalsy();
```

Comment:
no commented code please. the linter doesn't like double quotes so travis would probably fail so I would just delete this code or modify the linter if you really wanted this test case